src/components/Popover.vue

```diff
@@ -74,17 +75,23 @@ export default class Popover extends Vue {
     }
   }
 
-  mounted() {
+  async mounted() {
     // Save original parent before moving into body to use its position
     this.parent = this.$el.parentElement;
+    // Remove element from parent: it will be added to body when `setupPosition`
+    if (this.parent !== null) {
+      this.parent.removeChild(this.$el);
+    } else {
+      throw new Error('The popover has no parent!');
+    }
 
     // Get all scrollable parents
     const parents = [];
     let { parent } = this;
 
     while (parent !== document.body && parent !== null) {
       parents.push(parent);
-      parent = parent.parentElement;
+      parent = parent.parentElement as any; //FIXME: type any
```

Comment:
Will `Element` be a more appropriate type?

Reply:
`Element` is more generic than `HTMLElement`.
For example, it would be a `SVGElement` also.
I think this is why TS could disagree that `HTMLElement` is the type of a `node.parent`

Here is the associated MDN doc: https://developer.mozilla.org/en-US/docs/Web/API/Node/parentElement

Reply:
The issue comes not from `HTMLElement ` but from the fact parent can also be `null`... in the loop we ensure parent is not null, but parent.parentElement can be null and typescript does not like that...

Reply:
https://github.com/ToucanToco/weaverbird/pull/534/files#diff-46e1611babaddbb455a4f41bfb470d75R94